libtransmission/peer-mgr.cc

```diff
@@ -390,7 +390,7 @@ class tr_swarm
         return tor->unique_lock();
     }
 
-    [[nodiscard]] size_t countActiveWebseeds(uint64_t now) const noexcept
+    [[nodiscard]] uint16_t countActiveWebseeds(uint64_t now) const noexcept
```

Comment:
not sure why reducing the max possible number of webseeds.

By the way, line 1544, can we increase the size, from:
```
int8_t tr_peerMgrPieceAvailability(tr_torrent const* tor, tr_piece_index_t piece)
```
to at least:
```
int16_t tr_peerMgrPieceAvailability(tr_torrent const* tor, tr_piece_index_t piece)
```
otherwise it feels the max is 128 peers.